Dockerfile

```diff
@@ -35,15 +35,15 @@ RUN mkdir -p \
         /usr/share/man/man8 && \
     apt-get update && \
     apt-get install -y --no-install-recommends \
-        apt-transport-https postgresql-client netcat git curl  && \
+        apt-transport-https postgresql-client netcat build-essential git curl  && \
```

Comment:
This is needed since https://github.com/psf/black/pull/1047 switched black to use the regex library, which requires a compiler during build time.